pyproject.toml

```diff
@@ -153,6 +154,37 @@ allow_untyped_defs = true
 allow_incomplete_defs = true
 allow_untyped_calls = true
 
+[tool.ruff]
+line-length = 100
+lint.select = [
+    "ASYNC",
+    "B",
+    "C4",
+    "C90",
+    "E",
+    "F",
+    "FLY",
+    "PIE",
+    "S",
+    "W",
+]
+lint.ignore = [
+    "B017",
+    "B028",
+    "B904",
+    "E203",
+    "E501",
+]
```

Comment:
Some of these seem pretty useful. Why did you decide to ignore them instead of fixing them?

Reply:
Together those rules would require 65 fixes on the `main` branch even with `line-length=100`.  Modifying a dozen-plus files slows the review process.

My sense in the face of resistance to the tool was to land the tool in a pull request that did [not create too much change](https://github.com/PyCQA/isort/pull/2353#issuecomment-2637794904) and was therefore easy to review and merge before implementing more of the 800+ rules available.

Reply:
That's a lot.

Reply:
The majority of rules are not for this codebase.